src/app/styles/vendors/_slick-theme.scss

```diff
@@ -0,0 +1,194 @@
+@charset "UTF-8";
```

Comment:
It's an optional add-on that cleans up the default slick styles. Not the same as `slick.{css, less, scss}` that's included in the [bower package](https://github.com/kenwheeler/slick/blob/master/bower.json).